src/main/webapp/js/instructorCourseEdit.es6

```diff
@@ -469,8 +469,26 @@ function editFormRequest(e) {
     });
 }
 
+function enableOnclicks() {
+    $(document).on('click', '.addSectionLevelForInstructor', function () {
+        const index = $(this).attr('data-index');
+        const firstBlankSectionRowIndex = $(this).attr('data-first-blank-section-row-index');
```

Comment:
- `function ()` -> `(e) =>`
- `$(this)` -> `$(e.target)`
- `.attr('data-*')` -> `.data('*')`

Similarly for the one after this.